### DISCUSSION TITLE
advanced/friend-link

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#22] [Announcements]
# advanced/friend-link

VanBlog 支持 友情链接 展示， 你需要在 站点管理/系统设置/站点配置/布局设置 中开启 显示友情链接 选项。 友情链接展示 增删改查 进入后台 站点管理/数据管理/友情链接 即可，需要注意的是不同友链的名字必须不同，相同的话会覆盖掉之前的条目。 修改友情链接

https://vanblog.mereith.com/advanced/friend-link.html

**Comment by Xumin-Liang** [comment]
如何编辑友情链接页面的申领要求和本站信息等条目？能否在后台直接更改？
